### ISSUE TITLE
BlendArMocap v1.6.1 I can't install the Dependencies in Blender 3.6.0 and Blender 4.0.0.

### ISSUE THREAD

**Issue by beginner3D** [cgtinker/BlendArMocap#176]
I keep getting these errors when installing dependencies for Blender 3.6.0 and Blender 4.0.0.

I got the following error messages:
Installation of protobuf failed. Check system console output.
Installation of mediapipe failed. Check system console output.

**To Reproduce**
1. Installing the addon (no issue)
2. Click on 'Install Dependencies'
3. See errors: 'Installation of protobuf failed. Check system console output.' and 'Installation of mediapipe failed. Check system console output.'
4. Same error message for both Blender versions.

**Expected behavior**
I expected to install the addon and its dependencies correctly. But because I was unable to download the dependencies I can't use any of the features of the addon.

**Screenshots**
<img width="670" alt="Installing Dependencies" src="https://github.com/cgtinker/BlendArMocap/assets/126896643/c8fac05b-a4e7-4b31-9421-87004e91a712">
<img width="1280" alt="error message 1" src="https://github.com/cgtinker/BlendArMocap/assets/126896643/b73f0fe3-b341-44dd-8c19-04f868927274">
<img width="1280" alt="error message 2" src="https://github.com/cgtinker/BlendArMocap/assets/126896643/d464c1b4-1c8b-41e3-9b46-4b0dac4b69de">

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]: Microsoft Windows 11 Home
 - Blender Version: 3.6.0 and 4.0.0
 - BlendArMocap Version: v1.6.1 

**Additional context**
I find I have trouble installing dependencies in newer versions of Blender not only 3.6.0 and 4.0.0. I deleted the other Blender versions so I don't remember which other Blender versions I have trouble installing the dependencies. These are the two Blender versions I can say for sure that I can't install the dependencies.

**Comment by faresS69** [comment]
hey, i have recently installed BlendARMocap in my blender and it's working fine, in fact i didn't even installed dependencies because they were already installed, simply install open CV libraries in blender before starting,
First remove the BlendARMocap from your blender
Then follow this video starting from 3:00 to find how https://www.youtube.com/watch?v=O7nNO3FLkLU it shows you how to install openCV
Then add BlendARMocap again
Then let me know the results

**Comment by beginner3D** [comment]
@faresS69 yes it's working now in both versions of Blender. Thank you for the Youtube video link/
